test/mlir/conversion/onnx_to_torch/Concat/concat_axis_three.onnx.mlir

```diff
@@ -0,0 +1,13 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {}  {
+  func @main_graph(%arg0: tensor<1x4x12x12xf32>) -> tensor<1x12x12x24xf32> attributes {input_names = ["input"], output_names = ["4"]} {
+    %0 = "onnx.Constant"() {value = dense<"0xDEADBEEF"> : tensor<12x4x3x3xf32>} : () -> tensor<12x4x3x3xf32>
+    %1 = "onnx.Constant"() {value = dense<[-0.0787997097, 0.0934591516, -0.106071398, 0.105203271, 0.0158753395, -0.0936289653, -0.15529643, 0.165667564, 0.0695708394, -0.0372162871, 0.166498899, -0.0204498172]> : tensor<12xf32>} : () -> tensor<12xf32>
```

Comment:
nit: Please remove weights consistently. Also keep mlir folder names lowercase or uppercase.

Reply:
Hi Braun,
We have removed weights and changed mlir folder names to lowercase also.